doc/nrf/includes/vsc_build_and_run_nrf53.txt

```diff
@@ -0,0 +1,14 @@
+1. Open Visual Studio Code.
+#. Click on the :guilabel:`nRF Connect for VS Code` icon in the left navigation bar.
+#. Click on :guilabel:`Add an existing application`.
+#. In the prompt, navigate to the folder containing the sample you want to build, such as :file:`nrf/samples/bluetooth/peripheral_lbs`.
+   The selected application appears in the :guilabel:`Applications` panel.
+
+   An alternate option to open an application is to select the :guilabel:`Explorer` panel, right-click on the sample and select :guilabel:`Add Folder As Application` option.
+
+   .. note::
+
+     The sample folder must contain a :file:`prj.conf` file.
```

Comment:
This is no longer the case for NCS 2.0. The rule is now that it must contain a CMakeLists.txt file, although I don't think that info is worthy of getting a note in the documentation. Should just remove this note, IMO.